packages/now-cli/src/commands/dns/add.ts

```diff
@@ -117,3 +133,63 @@ export default async function add(
 
   return 0;
 }
+
+function readConfirmation(
+  output: Output,
+  msg: string,
+  domain: string,
+  argData: DNSRecordData
+) {
+  return new Promise(resolve => {
+    output.log(msg);
+    output.print(
+      `${table([getAddebleTableRow(domain, argData)], {
+        align: ['l', 'r', 'l'],
+        hsep: ' '.repeat(6)
+      }).replace(/^(.*)/gm, '  $1')}\n`
+    );
+    output.print(
+      `${chalk.bold.red('> Are you sure?')} ${chalk.gray('[y/N] ')}`
+    );
+    process.stdin
+      .on('data', d => {
+        process.stdin.pause();
+        resolve(
+          d
+            .toString()
+            .trim()
+            .toLowerCase() === 'y'
+        );
+      })
+      .resume();
+  });
+}
+
+function getAddebleTableRow(domain: string, argData: DNSRecordData) {
```

Comment:
I'm not sure what "Addeble" means. Is that a typo?

Reply:
It is.